app/views/connections/_form.html.erb

```diff
@@ -12,10 +12,6 @@
   <% end %>
 
   <div class="field">
-    <%= f.label :state %><br>
```

Comment:
State will automatically be set to pending once you create a connection. We shouldn't be able to set that when we create a connection